test/java/seedu/addressbook/commands/SearchEmailCommandTest.java

```diff
@@ -0,0 +1,60 @@
+package seedu.addressbook.commands;
+
+ import static org.junit.Assert.assertEquals;
+
+ import java.util.Arrays;
+ import java.util.Collections;
+ import java.util.HashSet;
+ import java.util.List;
+ import java.util.Set;
+
+ import org.junit.Before;
+ import org.junit.Test;
+
+ import seedu.addressbook.data.AddressBook;
+ import seedu.addressbook.data.exception.IllegalValueException;
+ import seedu.addressbook.data.person.ReadOnlyPerson;
+ import seedu.addressbook.util.TypicalPersons;
+
+public class SearchEmailCommandTest {
+	private AddressBook addressBook;
+	private TypicalPersons td;
+
+	@Before
+	public void setUp() throws Exception {
+		addressBook = new TypicalPersons().getTypicalAddressBook();
+		td = new TypicalPersons();
+	}
+
+	@Test
+	public void execute() throws IllegalValueException {
+		//invalid email - 0
+         assertSearchEmailCommandBehaviour(new String[]{"notanemail"}, Collections.emptyList());
+
+         //valid existing email - 1 result
+         assertSearchEmailCommandBehaviour(new String[]{"ab@gmail.com"}, Arrays.asList(td.amy));
+
+         //multiple emails: 2 results
+         assertSearchEmailCommandBehaviour(new String[]{"ab@gmail.com", "bc@gmail.com"},
+                 Arrays.asList(td.amy, td.bill));
```

Comment:
Minor style issue. If you want to put this on a new line, the same should also apply for the tests with similar style (e.g. the one directly below). This is just to ensure a consistent format.